### ISSUE TITLE
feat: Set Docker Resources Limits and Reservations for Celery  

### ISSUE THREAD

**Issue by null-ref-0000** [yogeshojha/rengine#1435]
### Is there an existing feature or issue for this?

- [X] I have searched the existing issues

### Expected feature

The purpose of this feature request is to set docker resource limits and reservations for celery. This will avoid potential out of memory issues due to celery consuming more memory than available  

### Alternative solutions

Set celery scaling configuration, but this however does not give you a precise option to control the cpu and memory resources utilized by a container. 

### Anything else?

https://docs.docker.com/reference/compose-file/deploy/#resources

**Comment by yogeshojha** [comment]
This will be an excellent addition, I am reviewing. I am also changing the compose files, they are messy and redundant. Will take this in together.


Thank you @null-ref-0000

**Comment by null-ref-0000** [comment]
@yogeshojha Sounds good, let me know if there anything else I can assist with regards to the compose files.

**Comment by yogeshojha** [comment]
@null-ref-0000 Definitely, once i make some progress tomorrow, I will add you as reviewer and you can guide me through.

Thank you for offering help.

**Comment by null-ref-0000** [comment]
@yogeshojha  need any help on this ? If you have any additional suggestions I can make them.
